Makefile.manual

```diff
@@ -4,6 +4,11 @@ FC ?= gfortran
 FFLAGS ?= -Wall -Wextra -Wimplicit-interface -fPIC -g -fcheck=all
 FYPPFLAGS ?=
 
+FYPPFLAGS += \
+             -DPROJECT_VERSION_MAJOR=$(shell cut -d. -f1 VERSION) \
+             -DPROJECT_VERSION_MINOR=$(shell cut -d. -f2 VERSION) \
+             -DPROJECT_VERSION_PATCH=$(shell cut -d. -f3 VERSION)
+
```

Comment:
Is the `shell cut` expected to run on Windows machines too?

Reply:
Can the manual makefile run on Windows? I always thought this is a Unix-only thing.

Reply:
The Quickstart Fortran installer comes with Make included, but I haven't tested it. I remember your advice was ninja + CMake is the better approach for Windows. Personally I'm fine leaving as it is until things break.

Reply:
Windows is particular in this case, the `shell` command might not work at all if no POSIX shell is available on the platform, but it might pickup the git-bash which is also included in the installer and could contain other small Unix command line utilities.

I can have a look into parsing files in pure make, which should be possible, but the result might look even more convoluted than the CMake version string splitting ;)

Reply:
`shell cut` fails for me on Windows cmd.exe and Msys2, even with `cut.exe` in PATH

```
fypp -DMAXRANK=4 stdlib_optval.fypp stdlib_optval.f90
common.fypp:4: error: exception occured when setting variable(s) 'PROJECT_VERSION' to '"{}.{}.{}".format(PROJECT_VERSION_MAJOR, PROJECT_VERSION_MINOR, PROJECT_VERSION_PATCH)' [FyppFatalError]
error: name 'PROJECT_VERSION_MAJOR' is not defined [NameError]
gnumake[1]: *** [Makefile.manual:74: stdlib_optval.f90] Error 1
```

Reply:
Found a solution using the make commands `file`, `subst` and `word`.